### ISSUE TITLE
Version Issue

### ISSUE THREAD

**Issue by GauravCreed** [jlmd/AnimatedCircleLoadingView#13]
Hi José Luis Martín.

great work you have done.but In my case your code is not working in Android Lolipop 5.1 but in below all version it will work perfectly,so please help me to sort out this  issue, i assume as a this is not a big deal to sort out this.

Thanks.

**Comment by jlmd** [comment]
It's working fine for me in 5.1 version too.
Can you give me more details, like a screenshot or something?

**Comment by hasanmumin** [comment]
yes android 5.1.1 does not working i have the same issue, animation is not determine. i debug code 

```
private void startAnimation() {
    if (getWidth() != 0 && getHeight() != 0) {
```

getWidth() also getHeight() returning 0.
